dix/dixfonts.c

```diff
@@ -1779,6 +1780,17 @@ DeleteClientFontStuff(ClientPtr client)
     }
 }
 
+int FillFontPath(x_rpcbuf_t *rpcbuf)
+{
+    for (int i = 0; i < num_fpes; i++) {
+        FontPathElementPtr fpe = font_path_elements[i];
+        /* write a pascal-string */
+        x_rpcbuf_write_CARD8(rpcbuf, fpe->name_length);
+        x_rpcbuf_write_CARD8s(rpcbuf, (CARD8*)fpe->name, fpe->name_length);
+    }
+    return num_fpes;
```

Comment:
The old code was also updating the global `font_path_string` var. Should we also do it here?